### DISCUSSION TITLE
add directive

### DISCUSSION THREAD

**Discussion by ibrahim-developper** [quasarframework/quasar#11190] [General - Components / Directives / etc] [answered]
**i want to add directive when scroll the page to do some animation.**

So my app use i18n and have i18n file in the boot folder i add my directive in this file

![i18n](https://user-images.githubusercontent.com/52491316/139920512-598c3454-9c1b-4e38-986d-b65ea89d6eec.png)

**and my directive file is** 
--------------------------------
![intersect](https://user-images.githubusercontent.com/52491316/139920479-876ffb15-fd00-41cc-b0a4-11c5e7c9f68f.png)

and last thing i put my directive to this h3 ( i try this also to other elements but not work)

`<h3 v-scrollanimation data-text="Popular Products">Popular Products</h3>`

**Expected behavior**
minimum this  console.log("register el",el) should see in the console bot nothing happen 

**Platform:**
Quasar Version: 2.2.2
@quasar/app Version: 3.1.8

Quasar mode:
  - [x] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

Tested on:
  - [x] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

**Comment by rstoenescu** [comment, marked as answer]
Please do not cross-post. https://github.com/quasarframework/quasar/issues/11187
